src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -82,6 +82,7 @@ final class MockTestStore: Store, PPAnalyticsData {
 	var clientMetadata: ClientMetadata?
 	var keySubmissionMetadata: KeySubmissionMetadata?
 	var testResultMetadata: TestResultMetadata?
```

Comment:
Maybe we could rename this property to make it clear that this is just for pcr? Keeping the underlying store as is fo course to keep compatibility.

```suggestion
	var pcrTestResultMetadata: TestResultMetadata?
```

Reply:
Not a fast change. Xcode refactoring tooling gave up on renaming this one.
I add a subtask to create a separate PR with this renaming.